Tasks/AzureRmWebAppDeployment/operations/WarDeploymentUtilities.ts

```diff
@@ -66,4 +66,4 @@ export async function HasWarExpandedSuccessfully(kuduService: Kudu, directoryWit
     // directory with same name as war file exists after deployment and if it existed before deployment, then the directory should contain content of new war file
     // which can be concluded if the modified time of the directory has changed.
     return filesAfterDeployment.some(item => { return item.name == warFileName && item.mime == "inode/directory" && (!directoryWithSameNameBeforeDeployment || item.mtime != directoryWithSameNameBeforeDeployment.mtime) });
-}
\ No newline at end of file
+}
```

Comment:
why this change? Similarly in parameterParser file

Reply:
removed extra spacing.

ParameterParser file I have moved into Operations folder which contains all the utility files.